doc/doc/basics/extractors.md

```diff
@@ -17,10 +17,15 @@ class contains different extractor *styles*:
 
 - Row-wise extractors:
   - [`json_extractor`](#json_extractor): highly flexible and compatible with
-  previous systems, but with limited performance 
+  previous systems, but with limited performance
```

Comment:
That'd be great!  The conflicts were mainly between ones trying to get into develop.  Well, master and develop is another thing we need to iron out, and we'll have a true "master" after this release with a clear doc on the branching/release policy.